Cpp_test/lib/ipc-shm-receive.cpp

```diff
@@ -125,12 +126,15 @@ void IPCShmReceive::map_shm()
 	{
 		throw std::runtime_error("ERROR: mmap64() data_ap received.");
 	}
+	this->shm_ptr->is_wait = true;
+	std::cout << this->shm_ptr->is_wait << std::endl;
```

Comment:
What is the purpose of is_wait and printing it out here? For what I can see this is the only place it is used in.

Reply:
Ah, this is the one that I try to make the program wait in another way, I forget to delete it, will delete it.